tests/acceptance/test_django.py

```diff
@@ -0,0 +1,26 @@
+import pathlib
+
+from django.db import connection
+
+
+def test_django_migrations_run_properly(db):
+    # At this point, with the db fixture, we have all migrations applied
+    with connection.cursor() as cursor:
+        cursor.execute("SELECT * FROM procrastinate_jobs")
+
+
+def test_no_missing_migration():
```

Comment:
This is a safeguard test to ensure we never add a `.sql` migration without its `.py` counterpart